classes/services/PKPSubmissionService.inc.php

```diff
@@ -473,7 +475,10 @@ public function getProperties($submission, $props, $args = null) {
 					$values[$prop] = $submission->getDateSubmitted();
 					break;
 				case 'dateStatusModified':
-					$values[$prop] = $submission->getDateStatusModified();
+					$values[$prop] = array(
+						'value' => $submission->getDateStatusModified(),
+						'label' => __('common.updated'),
+					);
```

Comment:
We prefer to pass locale strings like this in with the initial component. So that would be in the `i18n` object passed here: https://github.com/pkp/pkp-lib/blob/stable-3_1_2/controllers/list/submissions/PKPSubmissionsListHandler.inc.php#L119-L158

The `dateStatusModified` should always return just the date value. This keeps the UI and the API decoupled which makes it slightly easier to work with both.